docs/source/app-dev/app-arch.rst

```diff
@@ -81,13 +81,13 @@ your project, e.g. ``yarn add @daml/react``. Please explore the ``create-daml-ap
 to see the usage of these libraries.
 
 To make your life easy when interacting with the ledger, the DAML assistant can generate
-corresponding typescript data definitions for the data types declared in the deployed DAR.
+corresponding TypeScript data definitions for the data types declared in the deployed DAR.
 
 .. code-block:: bash
 
   daml codegen ts .daml/dist/<your-project-name.dar> -o daml.js
 
-This command will generate a typescript library for each DALF in you DAR.
+This command will generate a TypeScript library for each DALF in you DAR.
```

Comment:
Is generates a JavaScript library that also contains TypeScript typing for the generated JavaScript.